loklak.php

```diff
@@ -186,4 +186,54 @@ public function suggest($query="", $count=null, $order=null, $orderby=null, $sin
 			return json_encode($request, true);
 		}
 	}
+	
+	public function aggregations($query="", $since=null, $until=null, $fields=null, $limit=null) {
+		$this->requestURL = $this->baseUrl . '/api/search.json';
+		$this->query = $query;
+		$this->since = $since;
+		$this->until = $until;
+		$this->fields = $fields;
+		$this->$limit = $limit;
+		if ($query) {
+			$params = array('q'=>$this->query);
+			if ($since) {
+				$params['q'] = $params['q'] . ' since:'.$this->since;
+			}
+			if ($until) {
+				$params['q'] = $params['q'] . ' until:'.$this->until;
+			}
+			if ($fields) {
+				$field_string = "";
+				foreach ($this->fields as $field) {
+					$field_string += $field . ',';
+				}
+				$params['fields'] = $field_string;
+			}
+			if ($limit) {
+				$params['limit'] = $this->limit;
+			}
+			else { 
+				$this->limit = 6;
+				$params['limit'] = $this->limit;
+			}
+
+			$params['count'] = 0;
```

Comment:
Let us not hardcode the `count` value, let it be passed as one of the parameters, this is the `count` of the number of statuses to show, putting it 0 would result in `statuses : [ ]` in the resulting json which is okay in case of aggregation but we don't want to limit the user by forcing a 0 statuses on them.